package.json

```diff
@@ -6,25 +6,27 @@
   "repository": "https://github.com/GPII/gpii-handlebars",
   "scripts": {
     "test": "node tests/all-tests.js",
-    "postinstall": "grunt dedupe-infusion"
+    "postinstall": "grunt dedupe-infusion && bower install"
   },
   "dependencies": {
-    "express-handlebars": "1.1.0",
+    "express-handlebars": "1.2.2",
+    "gpii-express": "git://github.com/the-t-in-rtf/gpii-express#bc63bb7a8fe7e39b2aae20ce6c825daeb9cfcd58",
     "grunt": "~0.4.4",
     "grunt-shell": "0.6.4",
     "grunt-contrib-jshint": "~0.9.0",
     "grunt-jsonlint": "1.0.4",
     "grunt-gpii": "git://github.com/GPII/grunt-gpii.git#58ae055833f437a9e91baedb9f39a22c5fdd727f",
```

Comment:
Should be able to remove this dependency completely now - I see that the dedupe step has gone from the instructions